genie-core/src/main/java/com/netflix/genie/core/jpa/services/JpaBaseService.java

```diff
@@ -123,7 +126,9 @@ FileEntity createAndGetFileEntity(final String file) throws GenieException {
      * @return The tag entity that has been persisted in the database
      * @throws GenieException on error
      */
-    TagEntity createAndGetTagEntity(final String tag) throws GenieException {
+    TagEntity createAndGetTagEntity(
+        @NotBlank(message = "Tag name cannot be blank") final String tag
```

Comment:
is it really a "name"? Or is it a value? I'm never sure on this myself just wondering whether it's confusing cause the tag is what it is... it's a tag

Reply:
Yeah, i guess i could just say `Tag cannot be blank`